src/Microsoft.ApiDesignGuidelines.Analyzers/Core/AvoidEmptyInterfaces.cs

```diff
@@ -33,7 +35,17 @@ public abstract class AvoidEmptyInterfacesAnalyzer : DiagnosticAnalyzer
 
         public override void Initialize(AnalysisContext analysisContext)
         {
-            
+            analysisContext.RegisterSymbolAction(AnalyzeInterface, SymbolKind.NamedType);
+        }
+
+        private void AnalyzeInterface(SymbolAnalysisContext context)
```

Comment:
Looking at the FxCop implementation, it seems to not fire on interfaces that have method through their base interfaces. I,e `IFoo` itself is empty but `IFoo` derives from `IBar` which has some methods. Can you cover that case as well?